js/views/reuse.vue

```diff
@@ -194,6 +205,22 @@ export default {
     edit() {
       this.$go({ name: 'reuse-edit', params: { oid: this.reuse.id } })
     },
+    archive() {
+      this.reuse.archived = new Date().toISOString();
```

Comment:
Reacting late on this PR, but I think we would like to add a watch on `reuse.archived` to show a label when archived (similarly to datasets) for a feedback to the user on the action.